### ISSUE TITLE
bug: YouTube ReVanced does not start MIUI

### ISSUE THREAD

**Issue by ZeroDot1** [revanced/revanced-manager#351]
### Type

Other

### Bug description

After patching and installing Youtube Revanced does not start.

### Steps to reproduce

Completely normal use.

### Android version

12 MIUI 13 Latest

### Manager version

latest

### Target package name

Youtube Revanced

### Target package version.

latest

### Installation type

Non-root

### Patches selected.

All working patches.

### Device logs (exported using Manager settings).

```shell
none
```

### Installer logs (exported using Installer menu option) [unneeded if issue is not during patching].

```shell
none
```

### Screenshots or videos

0

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.
- [X] The issue is related solely to the ReVanced Manager

**Comment by manuel-rw** [comment]
I have the same issue

**Comment by Aunali321** [comment]
Fill proper details.

**Comment by its0x08** [comment]
Had same problem 
You just need to download the nodpi supported version of YouTube from apkmirror patch the apk and install it 
Thats the only way it works.
And disable the debug patch bcz crashes it on MIUI!

**Comment by Prashanth5533** [comment]
I have the same issue

**Comment by razza-guhl** [comment]
No issue here on MIUI 13, stock, non-root, bootloader locked.

I uninstalled official YouTube app before with adb per USB, as you cannot uninstall system-apps in GUI.

Using lastest supported YouTube Version 17.36.37 (https://www.apkmirror.com/apk/google-inc/youtube/youtube-17-36-37-release/youtube-17-36-37-2-android-apk-download/) and patched with reVanced with all available patches. 

I did everything as described in the Guide on discord (https://discord.com/channels/952946952348270622/1020635123894464512)
